XMLProjectIS/MetaData.cs

```diff
@@ -6,16 +6,15 @@
 //
 //    var welcome = Welcome.FromJson(jsonString);
 
-namespace QuickType
+namespace Movies
 {
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
     using System;
-    using System.Collections.Generic;
 
     using System.Globalization;
-    using Newtonsoft.Json;
-    using Newtonsoft.Json.Converters;
 
-    public partial class Welcome
+    public partial class MovieCollection
```

Comment:
Changed name of public class to “MovieCollection" which is relevant to the Project.